test/e2e/metamask-ui.spec.js

```diff
@@ -280,6 +282,14 @@ describe('MetaMask @no-mmi', function () {
     });
 
     it('renders the balance for the new token', async function () {
+      if (process.env.MULTICHAIN) {
+        return;
+      }
```

Comment:
I've noticed here that when we are in "Multichain" env. To be able to see the imported tokens you will need to have some "fiat balance". Either you have some ETH or you have balance in any other token. This works fine when we are on Mainnet.
However; when you are on a test network (in the case of this test: ganache) the tokens you import are worthless; so in order to be able to see the imported tokens you will need to have some ETH. 
For some reason, the balance of the wallet in this test =0 (with multichain) and 1000 with not multichain... 
I ve also noticed that we are skipping doing "send" tests if `(process.env.MULTICHAIN)` because of the balance issue...